parl/remote/grpc_heartbeat/heartbeat_client.py

```diff
@@ -22,22 +22,33 @@
 
 
 class HeartbeatClientThread(threading.Thread):
-    def __init__(self, heartbeat_server_addr, heartbeat_exit_callback_func):
+    def __init__(self,
+                 heartbeat_server_addr,
+                 heartbeat_exit_callback_func,
+                 exit_func_args=(),
+                 exit_func_kwargs={}):
         """Create a thread to run the heartbeat client.
 
             Args:
                 heartbeat_server_addr(str): the address of the heartbeat server.
                 heartbeat_exit_callback_func(function): A callback function, which will be called after the 
-                                                        heartbeat exit. There should be no arguments in the 
-                                                        function.
+                                                        heartbeat exit.
+                exit_func_args(tuple): the argument tuple for the heartbeat_exit_callback_func invocation. Defaults to ().
```

Comment:
calling the heartbeat_exit_callback_func.

Reply:
fixed